runtime/doc/lsp.txt

```diff
@@ -2131,6 +2131,35 @@ stop({bufnr}, {client_id})                    *vim.lsp.semantic_tokens.stop()*
       • {client_id}  (`integer`) The ID of the |vim.lsp.Client|
 
 
+==============================================================================
+Lua module: vim.lsp.document_color                        *lsp-document_color*
+
+enable({enable}, {filter})                   *vim.lsp.document_color.enable()*
```

Comment:
@justinmk LMK what you think of this API.

Reply:
👍 this matches the pattern from `vim.lsp.inlay_hint`.

Maybe in the future we might think about a `vim.lsp.enable({features})` interface instead of these per-feature modules, but each of these modules (inlay_hint, document_color) is non-trivial, so it's understandable that they have their own modules, I suppose...